packages/typespec-client-generator-core/src/decorators.ts

```diff
@@ -571,8 +571,11 @@ export function listOperationsInOperationGroup(
   return operations;
 }
 
-interface CreateSdkContextOptions {
+const defaultDecoratorsWhiteList = ["TypeSpec\\.Xml\\..*"];
```

Comment:
I would prefer to put the default whitelist into a seperate config file to be easier to find and update. What do others think? I'm open to other opinions.